analyses/copy_number_consensus_call/src/scripts/compare_variant_calling_updated.py

```diff
@@ -42,92 +42,55 @@
 
 
 
-## Define parser for the input file
-parser = argparse.ArgumentParser(description="""This script takes in 3 bed files, each from one of the 3 callers
-                                                 and find common CNVs between two files at a time.""")
-parser.add_argument('--manta', required=True,
-                    help='path to the manta file')
-parser.add_argument('--cnvkit', required=True,
-                    help='path to the cnvkit file')
-parser.add_argument('--freec', required=True,
-                    help='path to the freec file')
-parser.add_argument('--manta_cnvkit', required=True,
-                    help='path of the output consensus between manta and cnvkit')
-parser.add_argument('--manta_freec', required=True,
-                    help='path of the output consensus between manta and freec')
-parser.add_argument('--cnvkit_freec', required=True,
-                    help='path of the output consensus between cnvkit and freec')
-parser.add_argument('--sample', default='no_sample',
-                    help='sample name to use in the file')
-
-args = parser.parse_args()
-
-## Put the input and output file paths into their own lists that is to be iterated over
-## This order is important
-list_of_files = [args.manta, args.cnvkit, args.freec]
-list_of_output_files = [args.manta_cnvkit, args.manta_freec, args.cnvkit_freec]
-
-## Create a list to store the content of the 3 caller CNV files
-list_of_list = []
-
-
-## Loop through the file paths and read in their contents to store in a list
-for i in list_of_files:
+########################## DEFINE FUNCTIONS ###############################
 
+## Function to read content of input files
+def read_input_file(input_file):
     ## Check to see if the CNVs bed file is empty, if so, make an empty variable
```

Comment:
Before the comments that start to describe individual elements, you should add an overall description of the function here. The python convention is to use a triple quoted line or lines immediately below the def line that describes what the function does, along with expected inputs and outputs (and their types). See more here: https://www.python.org/dev/peps/pep-0257/